packages/test/local-server-stress-tests/src/test/localServerStress.spec.ts

```diff
@@ -83,7 +83,9 @@ function makeGenerator(): AsyncGenerator<StressOperations, LocalServerStressStat
 export const saveFailures = { directory: path.join(_dirname, "../../src/test/results") };
 export const saveSuccesses = { directory: path.join(_dirname, "../../src/test/results") };
 
-describe("Local Server Stress", () => {
+// TODO (AB#33713): these tests are a bit flaky; in the pipeline we've seen seeds 43 and 44 fail with
+// errors that might represent bugs in the underlying DDSes. Disabling for now.
+describe.skip("Local Server Stress", () => {
```

Comment:
can we only skip the seed and not the whole test, just like for seed 28 below. i think only 44 is problematic

Reply:
Sure. I'll skip 43 and 44 then, since I've seen both fail in the pipeline (once each).